core/backend/src/CloudSqlite.ts

```diff
@@ -103,31 +105,33 @@ export namespace CloudSqlite {
   }
   export interface ContainerProps {
     /** The type of storage provider. */
-    storageType: "azure" | "google";
+    readonly storageType: "azure" | "google";
```

Comment:
these member should always have been readonly. But we now store ContainerProps in memory to identify containers. This change makes it so they're immutable and therefore reliable identifiers.